src/plugin/cesium/cesium.js

```diff
@@ -540,68 +518,48 @@ plugin.cesium.hasWorldTerrain = function() {
   return false;
 };
 
-
 /**
  * Enable the Cesium World Terrain provider, if configured.
  */
-plugin.cesium.enableWorldTerrain = function() {
-  const map = os.MapContainer.getInstance();
+const enableWorldTerrain = function() {
+  const map = MapContainer.getInstance();
   const renderer = map.getWebGLRenderer();
   if (renderer) {
     const supported = renderer.getSupportedTerrainProviders();
     const worldTerrain = supported.find((p) => p.type === os.map.terrain.TerrainType.ION);
```

Comment:
Still using a global reference for `TerrainType` here.

Reply:
I started converting some globals, but we should write a transform to do that for us in an intelligent manner using the `deps.js` file. Otherwise it's going to add a large amount of tedious extra work across all of OpenSphere's sources.